src/components/Hero.astro

```diff
@@ -1,45 +1,83 @@
 ---
-import { Icon } from 'astro-icon'
+import { Icon } from "astro-icon";
 
-const { src = '/astronaut-hero-img.webp' } = Astro.props
+const { src = "/astronaut-hero-img.webp" } = Astro.props;
 ---
 
-<section class="hero my-24">
+<div class="background"></div>
+<section class="hero my-19">
   <div class="container">
-    <div class="grid grid-cols-1 items-center gap-24 lg:grid-cols-2">
+    <div class="items-center gap-24 lg:grid-cols-2">
       <div class="flex flex-col items-center gap-8 md:items-start">
         <h1 class="text-center text-6xl md:text-left lg:text-8xl">
-          <slot><span class="text-gradient">Accessible</span> Starter for Astro</slot>
+          <slot
+            ><span class="text-gradient"
+              >Breaking down the <br /> barriers of corporate America</span
+            >
+            <br />
+            <span class="accent">One topic at a time</span>
+          </slot>
         </h1>
         <div class="flex flex-col gap-3 min-[500px]:flex-row">
-          <a class="button has-icon" href="https://github.com/markteekman/accessible-astro-starter">
+          <a
+            class="button has-icon"
+            href="https://github.com/CareerCatalyst/Career-Catalyst"
+          >
             <Icon pack="ion" name="star-outline" />
             Star on GitHub
           </a>
-          <a
-            class="button has-icon color-secondary"
-            href="https://github.com/markteekman/accessible-astro-starter/blob/main/README.md"
-          >
+          <a class="button has-icon color-secondary">
             <Icon pack="ion" name="bookmark-outline" />
-            Read the Docs
+            Newsletter
           </a>
         </div>
       </div>
-      <img class="hidden lg:block" src={src} alt="" decoding="async" />
+      <!-- <img class="hidden lg:block" src={src} alt="" decoding="async" /> -->
     </div>
   </div>
 </section>
 
 <style lang="scss">
   .text-gradient {
-    background: linear-gradient(315deg, var(--primary-200) 25%, var(--secondary-500));
-    background-clip: border-box;
-    -webkit-background-clip: text;
-    -webkit-text-fill-color: transparent;
+    // background: linear-gradient(
```

Comment:
do we still need the commented out part?

Reply:
Not really was more experimenting with it and seeing how it looks